### ISSUE TITLE
Change map & mode title in the server list

### ISSUE THREAD

**Issue by FredericaBernkastel** [FredericaBernkastel/codmw3-server-DGAdmin-plugin#14]
This is a nice feature from SinAdmin. I wish to add it in DG.

![](https://raw.githubusercontent.com/FredericaBernkastel/codmw3-server-DGAdmin-plugin/master/doc/img/002.png)

**Comment by carck007** [comment]
yes i hope that so

**Comment by FredericaBernkastel** [comment]
The structure we're searching for

![](https://raw.githubusercontent.com/FredericaBernkastel/codmw3-server-DGAdmin-plugin/master/doc/img/003.png)

So pattern would be:
**`67 6E 5C 49 57 35 5C 67 74 5C`**

Several occurencies will be found, but only one of them is not latched.

**Possible algorithm:**

1. Scan virtual memory up to 0x8000000 (128M) for the pattern
2. Test each occurency for latchness by writing single byte inside the structure
3. Occurency, that not latched, will be correct one that we're looking for
4. Read the structure as ASCII-String
5. We can modify the string, and write it back. But note about limits:

```
mode name max length: 15
map name max length:  31
structure max length: 128 bytes
```

**Comment by FredericaBernkastel** [comment]
`Closed by commit `0e37349`.`
